### PR TITLE
Add a delete_indices helper to delete all indices during tests.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: conf/tests.ini [hunk 1]
```diff
@@ -11,6 +10,9 @@ elasticsearch.hosts = localhost:9200
 # backend.db_host = http://localhost:5984/
 # backend.db_name = daybed-test
 
+elasticsearch.hosts = localhost:9200
+elasticsearch.indices_prefix = daybed_tests
```
Comment (leplatrem) [inline conf/tests.ini:14]: not necessary I guess (a lot of default values everywhere)

File: conf/development.ini [hunk 2]
```diff
@@ -3,6 +3,7 @@ use = egg:daybed
 # Backend configuration
 daybed.backend = daybed.backends.redis.RedisBackend
 elasticsearch.hosts = localhost:9200
+elasticsearch.indices_prefix = daybed
```
Comment (leplatrem) [inline conf/development.ini]: not necessary (default value)

File: daybed/indexer.py [hunk 3]
```diff
@@ -8,21 +8,22 @@
 
 class ElasticSearchIndexer(object):
 
-    def __init__(self, hosts):
+    def __init__(self, hosts, prefix='daybed'):
```
Comment (leplatrem) [inline daybed/indexer.py]: no default necessary here

File: conf/development.ini (hunk 2 above)
Comment (Natim) [inline conf/development.ini]: ok

File: conf/tests.ini (hunk 1 above)
Comment (Natim) [inline conf/tests.ini:14]: We shouldn't use the default for tests.

File: conf/development.ini (hunk 2 above)
Comment (Natim) [inline conf/development.ini]: Also I think if people uses it as a sample, they should be aware of this `indices_prefix`

**Comment by Natim** [comment]
Updated

**Pull request merged**